utopia-remix/.env.sample

```diff
@@ -1,26 +1,23 @@
-APP_ENV="local"
-BACKEND_URL="http://127.0.0.1:8002"
-CORS_ORIGIN="http://localhost:8000"
-DATABASE_URL="postgres://<username>:postgres@localhost:5432/utopia"
-REACT_APP_EDITOR_URL="http://localhost:8000"
+# APP_ENV="local"
+# BACKEND_URL="http://127.0.0.1:8002"
```

Comment:
Since we can now locally use `localhost` here again (I forget why we couldn't before), and this is just `EXPRESS_PROXY_TARGET_HOST` plus `EXPRESS_PROXY_TARGET_PORT`, I'd say we don't need a separate variable for these - we should either have one for `BACKEND_URL` and calculated the proxy target from that, or vice versa